MinecraftClient/Mapping/Material2Tool.cs

```diff
@@ -0,0 +1,706 @@
+                Material.SprucePressurePlate,
+                Material.SpruceSign,
+                Material.SpruceSlab,
+                Material.SpruceStairs,
+                Material.SpruceTrapdoor,
+                Material.SpruceWallSign,
+                Material.SpruceWood,
+                Material.StrippedAcaciaLog,
+                Material.StrippedAcaciaWood,
+                Material.StrippedBirchLog,
+                Material.StrippedBirchWood,
+                Material.StrippedCrimsonHyphae,
+                Material.StrippedCrimsonStem,
+                Material.StrippedDarkOakLog,
+                Material.StrippedDarkOakWood,
+                Material.StrippedDarkOakWood,
+                Material.StrippedJungleLog,
+                Material.StrippedJungleWood,
+                Material.StrippedOakLog,
+                Material.StrippedOakWood,
+                Material.StrippedSpruceLog,
+                Material.StrippedSpruceWood,
+                Material.StrippedWarpedHyphae,
+                Material.StrippedWarpedStem,
+                Material.TrappedChest,
+                Material.Vine,
+                Material.WarpedButton,
+                Material.WarpedDoor,
+                Material.WarpedFence,
+                Material.WarpedFenceGate,
+                Material.WarpedHyphae,
+                Material.WarpedPlanks,
+                Material.WarpedPressurePlate,
+                Material.WarpedSign,
+                Material.WarpedSlab,
+                Material.WarpedStairs,
+                Material.WarpedStem,
+                Material.WarpedTrapdoor,
+                Material.WarpedWallSign,
+                Material.WhiteBanner,
+                Material.WhiteWallBanner,
+                Material.YellowBanner,
+                Material.YellowWallBanner
+        });
+        // Every block a shear can mine.
+        private readonly List<Material> shears = new List<Material>(new Material[]
+        {
+                Material.AcaciaLeaves,
+                Material.BirchLeaves,
+                Material.BlackWool,
+                Material.BlueWool,
+                Material.BrownWool,
+                Material.Cobweb,
+                Material.CyanWool,
+                Material.DarkOakLeaves,
+                Material.GrayWool,
+                Material.GreenWool,
+                Material.JungleLeaves,
+                Material.LightBlueWool,
+                Material.LightGrayWool,
+                Material.LimeWool,
+                Material.MagentaWool,
+                Material.OakLeaves,
+                Material.OrangeWool,
+                Material.PinkWool,
+                Material.PurpleWool,
+                Material.RedWool,
+                Material.SpruceLeaves,
+                Material.WhiteWool,
+                Material.YellowWool,
+        });
+        // Every block that is mined with a sword.
+        private List<Material> sword = new List<Material>(new Material[]
```

Comment:
Missing `readonly` here.
```suggestion
        private readonly List<Material> sword = new List<Material>(new Material[]
```